### DISCUSSION TITLE
Variable Fonts with Roman and Italic, what to do ?

### DISCUSSION THREAD

**Discussion by SolennBor** [fonttools/fontbakery#3824] [General]
We have 2 Variable Fonts for the same Family, one for Romans and one for Italics, they should be style linked.
Fontbakery raise `Fails` about the italic VF:
- ‘fvar’:
    - `The “wght” axis coordinate of the “Regular” instance must be 400. Got None instead. [code: wght-not-400]` obviously the font doesn’t have Regular but Italic instead
    -  `The “wght” axis coordinate of the “Bold” instance must be 700. Got None instead. [code: wght-not-700]` same thing, no Bold but Bold Italic instead
- ‘STAT’: `The following AxisValue entries on the STAT table should not contain “Italic”: [‘nameID 258: Light Italic’, ‘nameID 259: Regular Italic’, ‘nameID 260: Bold Italic’, ‘nameID 261: ExtraBold Italic’] [code: bad-italic]`
Q: Should we add an `ital` axis with `default=“1" maximum=“1” minimum=“1"` for specifying the font is italic even though it doesn’t vary on this axis? (similarly the Roman VF could have this `ital` axis set with `default=“0” maximum=“0" minimum=“0”`)
Is there a standard way at solving this issue that must be quite common indeed?
Thanks for your input.

**Comment by felipesanches** [comment]
It looks like a Font Bakery bug. Are you able to provide sample font files so that I can try to replicate the problem locally and then try to fix it?

**Reply by SolennBor** [reply]
Thanks for your reply, I sent you an email with the files!

**Comment by justvanrossum** [comment]
The issue could have been the same as the one that was fixed by https://github.com/fonttools/fonttools/pull/2684

**Reply by justvanrossum** [reply]
In other words: we're having some trouble reproducing it today with an up-to-date toolchain.

**Reply by felipesanches** [reply]
Thanks. I'll take a look at that when I start inspecting the files sent to me by @SolennBor last week. Sorry for not having addressed this yet. It's been a pretty busy month for me here.

**Reply by justvanrossum** [reply]
No worries, and thanks either way. Let's drop the issue for now, and we'll get back to it if we manage to reproduce it after all. (The "we" is me and @SolennBor.)

**Reply by felipesanches** [reply]
Oh, I see. It was not immediately clear to me that you were both working together. Do you still want me to inspect the files? Or can I remove that task from my TO-DO list?

**Reply by justvanrossum** [reply]
Sorry for the confusion. Yes, please remove that task. If it comes up again we'll get back here with a reproducer.

**Reply by felipesanches** [reply]
Cool, thanks!

**Comment by BlackFoundry** [comment]
When the VF is purely Italic there is no 'Regular' or 'Bold' styles (but 'Italic' and 'Bold Italic' instead), and it seems FontBakery gets confused with these checks in particular, and fails:
`com.google.fonts/check/varfont/regular_wght_coord`
`com.google.fonts/check/varfont/bold_wght_coord`
`com.google.fonts/check/varfont/regular_wdth_coord`

Shouldn't it behave differently in such a situation?

**Reply by m4rc1e** [reply]
I think I fixed this in https://github.com/googlefonts/fontbakery/pull/3834. I don't think it has made its way into a fontbakery release yet.

**Reply by BlackFoundry** [reply]
oh great! thanks for the reply.
